### PR TITLE
bugs fixed

### PR DESCRIPTION
**Pull request by JalandharBandari**
bugs fixed

### CODE REVIEW

File: src/components/RoomsBoys/RoomsBoys.jsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const RoomsBoys = () => {
     const value = event.target.value;
     const re = /^[0-9\b]+$/; // Regular expression to allow only numbers
 
-    if (value === '' || re.test(value)) {
+    if (value === '' || re. test(value)) {
```
Comment (TejaBonam) [inline src/components/RoomsBoys/RoomsBoys.jsx:30]: Is this needed?

File: src/components/RoomsBoys/RoomsBoys.jsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const RoomsBoys = () => {
...
         switch(event.target.name) {
```
Comment (TejaBonam) [inline src/components/RoomsBoys/RoomsBoys.jsx:31]: Is this needed @JalandharBandari

File: src/pages/Login/Login.css [hunk 3]
```diff
@@ -56,7 +56,7 @@ text-align: center;
 
 /* .down-image{
 /* height: auto; */
- */
+
```
Comment (TejaBonam) [inline src/pages/Login/Login.css:59]: Check this one also @JalandharBandari

File: src/components/RoomsBoys/RoomsBoys.jsx (hunk 2 above)
Comment (JalandharBandari) [reply src/components/RoomsBoys/RoomsBoys.jsx:31]: Yes this is required but remove space before test(value)

File: src/pages/Login/Login.css (hunk 3 above)
Comment (JalandharBandari) [reply src/pages/Login/Login.css:59]: it was a type error please remove @TejaBonam

**Pull request merged**